src/metabase/analytics/stats.clj

```diff
@@ -141,8 +141,11 @@
    :embedding_app_origin_set             (boolean
                                           #_{:clj-kondo/ignore [:deprecated-var]}
                                           (setting/get :embedding-app-origin))
+   ;; We no longer add "localhost:*" as a default origin as of Metabase 56, as it is always allowed,
+   ;; but we still filter it out in stats for compatibility with migrated instances.
    :embedding_app_origin_sdk_set         (boolean (let [sdk-origins (setting/get :embedding-app-origins-sdk)]
-                                                    (and sdk-origins (not= "localhost:*" sdk-origins))))
+                                                    (and (not (str/blank? sdk-origins))
+                                                         (not= "localhost:*" sdk-origins))))
```

Comment:
Stats should consider both `localhost:*` and `""` (a blank string) to be unset origins, the former for previous instances and the latter for new instances. For compatibility reasons.